inc/namespace.php

```diff
@@ -130,26 +141,59 @@ function get_config() {
 }
 
 /**
- * Load the Object Cache dropin.
+ * Load the object cache.
+ *
+ * Check the object caching configuration and load either memcached
+ * or redis as appropriate.
+ *
+ * @deprecated 1.1.0 Object caching setup moved to dedicated functions.
  */
 function load_object_cache() {
-	wp_using_ext_object_cache( true );
+	if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+		trigger_error(
+			sprintf(
+				__( '%1$s is <strong>deprecated</strong> since version %2$s! Use %3$s instead.' ),
+				__FUNCTION__,
+				'1.1.0',
```

Comment:
I want to double check this but pretty sure we won't be doing any 1.x.x releases that aren't RC versions. Because it's a bug fix and backwards compatible we'll do this as a patch release.
```suggestion
				'1.0.1',
```